### PR TITLE
Add explanation on how to open a drawer

### LINKED ISSUE #10519: Update wasm page for stable
### Page URL

https://docs.flutter.dev/platform-integration/web/wasm

### Page source

_No response_

### Describe the problem

We should update this page to mention that this is now stable

### Expected fix

- [ ]  Mention it's stable.
- [ ] Update details on hosting wasm. Should include the headers we mention in dhttpd – but for any host. Including details for firebase would be good.
- [ ] Explain that some hosters (GitHub) WON'T work because they can't set custom headers. but Firebase DOES!
- [ ] Mention that you NEED to update to the latest bootstrap logic - https://docs.flutter.dev/platform-integration/web/bootstrapping

### Additional context

_No response_

### I would like to fix this problem.

- [ ] I will try and fix this problem on docs.flutter.dev.

### PR DESCRIPTION
**Pull request by sudot**

### CODE REVIEW

**Comment by domesticmouse** [comment]
> Resolve #10519

I don't see how this PR is related to that issue.

File: src/content/cookbook/design/drawer.md [hunk 1]
```diff
@@ -110,7 +110,37 @@ Drawer(
 );
 ```
 
-## 4. Close the drawer programmatically
+## 4. Open the drawer programmatically
+
+Typically, you don't need to write any code to open a `drawer`,
+Because when leading is `null`, the default implementation in `AppBar` is `DrawerButton`.
```
Comment (sfshaza2) [inline src/content/cookbook/design/drawer.md:116]: When leading <WHAT> is null?

**Review by sfshaza2 — CHANGES_REQUESTED** [review]
Hi, @farcaller!

- This is a great improvement, thx! However, the build tests are failing. When you change the code, you have to re-run the refresh code excerpts script. (This is described in the site README.)
- Also, I found one typo. Can you fix that?
- @craiglabenz, can you otherwise review the code in this PR?

**Review by craiglabenz — CHANGES_REQUESTED** [review]
Thanks, @sudot. This LGTM once the build is passing. One of the problems is formatting (should be an easy fix), and the other error message can be found here: https://github.com/flutter/website/actions/runs/9013474562/job/24764429256?pr=10548

**Review by craiglabenz — APPROVED** [review]
LGTM

**Comment by craiglabenz** [comment]
It looks like the staging build failed because my (personal Gmail) account lacks permissions in Cloud Build for this project.

IMO we can either safely merge or someone whose GitHub account is tied to a Google Cloud account with sufficient permissions can re-merge in `main` to trigger the job again.

**Review by sfshaza2 — APPROVED** [review]
Thanks, @sudot and @craiglabenz!

**Pull request merged**